examples/target-aquisition-and-confirmation.rs

```diff
@@ -143,7 +153,28 @@ fn main() {
         .build();
     let expected_target_ecef = Coordinate::<Ecef>::from_wgs84(&expected_target_wgs84);
 
-    println!("\nChecking calculated plane position given expected plane position");
+    let expected_ground_distance_to_target = Length::new::<meter>(22515.102);
```

Comment:
nit: let's leave a comment here saying that these were computed manually and committed to ensure no regression